src/widget.jl

```diff
@@ -7,26 +7,28 @@ mutable struct Widget{T, S} <: AbstractWidget{T, S}
     output::AbstractObservable{S}
```

Comment:
Maybe this can be `Observable{S}`? In practice it seems like the inner constructor only accepts observables anyway.

Reply:
Ah yes I was wondering about that. It should help with stability.